lib/modules/apostrophe-areas/lib/helpers.js

```diff
@@ -188,6 +188,55 @@ module.exports = function(self, options) {
 
     plaintext: function(within, options) {
       return self.plaintext(within, options);
+    },
+    
+    widgetControlGroups: function() {
```

Comment:
You have a weird combination of my new widgetControlGroups stuff, which is in workflow-accommodations-1, with the old widgetControls.html code that had all the individual controls hardcoded. I suggest reopening this PR against workflow-accommodations-1 (to be merged into it).